components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -0,0 +1,150 @@
+    pendingPlatformFees,
+    pendingPlatformTips,
+    pendingHostFeeShare,
+    platformTips,
+    hostFees,
+    platformFees,
+  } = hostMetrics;
+  const currentAmount = totalMoneyManaged.valueInCents;
+  const projectedAmount =
+    totalMoneyManaged.valueInCents +
+    pendingPlatformFees.valueInCents +
+    pendingPlatformTips.valueInCents +
+    pendingHostFeeShare.valueInCents;
+  const totalCollectiveFunds =
+    totalMoneyManaged.valueInCents - platformTips.valueInCents - platformFees.valueInCents - hostFees.valueInCents;
+  const totalHostFunds = hostFees.valueInCents;
+  return (
+    <StyledCard borderColor="#46347F">
+      <Container pl={27} pr={24} pt={16} pb={16} backgroundColor="#F6F5FF">
+        <P minHeight="16px" fontSize="12px" fontWeight="500" lineHeight="16px" textTransform="uppercase">
+          <Dollar size={14} color="#1300AB" />
+          <FormattedMessage id="Host.Metrics.TotalMoneyManages" defaultMessage="Total Money Managed" />
+        </P>
+        <Flex flexWrap="wrap" mt={12} mb={14}>
+          <Span fontSize={18} fontWeight="500">
+            {formatCurrency(currentAmount, currency)}
+          </Span>
+          <Span fontSize={17} fontWeight="500" lineHeight="20px" ml="8px" mr="8px">
+            /
+          </Span>
+          <Span fontSize={15} fontWeight="500" lineHeight="25px">
+            {formatCurrency(projectedAmount, currency)}
+          </Span>
+          <Span fontSize={12} fontWeight="500" lineHeight="27px" ml="8px">
+            <FormattedMessage id="TotalMoneyManagedSection.projected" defaultMessage="Projected" />
+          </Span>
+        </Flex>
+        <P minHeight="12px" fontSize="11px" fontWeight="700" lineHeight="12px" textTransform="uppercase">
+          <Span style={{ verticalAlign: 'middle' }}>
+            <FormattedMessage
+              id="TotalMoneyManagedSection.subHeading"
+              defaultMessage="My Organization and My Collectives"
+            />
+          </Span>
+          <Span ml={1}>
+            <StyledTooltip
+              content={() => (
+                <FormattedMessage
+                  id="TotalMoneyManagedSection.titleTooltip"
+                  defaultMessage="Total amount held in your bank account for the Host and its Collectives."
+                />
+              )}
+            >
+              <InfoCircle size={14} />
+            </StyledTooltip>
+          </Span>
+        </P>
+        <Container mt={18} mb={12}>
+          <FundAmounts>
+            <TotalFundsLabel minWidth="210px">
+              <P>
+                <FormattedMessage
+                  id="TotalMoneyManagedSection.totalCollectiveFunds"
+                  defaultMessage="{totalCollectiveFunds} | Collectives"
+                  values={{
+                    totalCollectiveFunds: (
+                      <Span fontWeight="700">{formatCurrency(totalCollectiveFunds, currency)}</Span>
+                    ),
+                  }}
+                />
```

Comment:
Here, having the variable in the string makes it more complex and doesn't bring any value. Unless the separator character is going to change across languages, the simplest solution (both for developers and translators) is to do:

```suggestion
                <Span fontWeight="700">{formatCurrency(totalCollectiveFunds, currency)}</Span> |{' '}
                <FormattedMessage id="pricingTable.row.collectives" defaultMessage="Collectives" />
```

And the string already exists here, so that's one less duplication to work on for translators.

Reply:
Thanks, makes sense. I've corrected it. 👍🏽